css/styles.css

```diff
@@ -70,11 +70,196 @@ body {
 }
 
 /*=== media queries ===*/
-@media screen and (min-width: 401px) {
+@media only screen and (max-width: 400px) {
```

Comment:
these media queries look great! Thank you for catching the oversight with our previous max-width 400px  and min-width 401px. This makes for a much smoother transition.